components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2863,6 +2874,7 @@ where
     pub entries: SmallVec<[CachedEntries; 1]>,
     pub entries_size: usize,
     pub cbs: Vec<Proposal<S>>,
+    pub buckets: Option<Arc<BucketMeta>>,
```

Comment:
What's the purpose?

Reply:
`Option` for indicating whether the buckets are initialized, `Arc` for making it be cloned with lower cost.

Reply:
What I mean is why apply fsm needs to know buckets meta?

Reply:
Need to collect the flow of buckets when writing keys.